x-pack/plugins/observability_solution/logs_explorer/public/components/data_source_selector/constants.tsx

```diff
@@ -88,7 +88,7 @@ export const noDataRetryLabel = i18n.translate(
 );
 
 export const tryEsql = i18n.translate('xpack.logsExplorer.dataSourceSelector.TryEsql', {
-  defaultMessage: 'Try ES|QL',
+  defaultMessage: 'Language: ES|QL',
```

Comment:
**question:** this text is for the CTA on the Data Source Selector from Logs Explorer, is it expected to change to a less actionable label?

Reply:
Yes it is the same text we are using in Discover. This text was given to me but I can pass your concern.

Reply:
I don't want this to be a blocker, it's mostly a personal opinion on how catchy the label is :)

Reply:
Marco, I passed your comment. It seems that we are ok now with it as we are planning on a redesign. It is a temporary solution for GA.